documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -60,7 +60,7 @@ Raspberry Pi 400 has a dedicated power button which operates even if the process
 
 On Raspberry Pi 5 this places the PMIC in `STANDBY` mode where all outputs are switched off. There is no need to set `WAKE_ON_GPIO` and pressing the xref:raspberry-pi.adoc#power-button[dedicated power button] will boot the device.
 
-Default: `0`
+Default: `1` on Compute Modules since Compute Module 5 and Keyboard models since Raspberry Pi 500; otherwise `0`
```

Comment:
```suggestion
Default: `1` on Compute Modules since 5 and Keyboard models since 500; otherwise `0`
```

Reply:
@timg236 since 400 has a power button, I assume the keyboard models adopted this default in the 400 model.